c4200m_get_target_sentences.py

```diff
@@ -5,44 +5,75 @@
 
 from absl import app
 import tensorflow_datasets as tfds
+import multiprocessing as mp
+import re
 
 LOGGING_STEPS = 100000
+BATCH_SIZE = 64
 
+def rreplace(string, old, new):
```

Comment:
Slightly hard to parse. Please add a short docstring describing what this function is doing.